tsl/src/bgw_policy/policies_v2.c

```diff
@@ -681,6 +682,10 @@ policies_show(PG_FUNCTION_ARGS)
 								  SHOW_POLICY_KEY_RETENTION_INTERVAL,
 								  &(job->fd.schedule_interval));
 		}
+		else
+			ereport(ERROR,
+					(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+					 errmsg("\"%s\" unsupported proc", NameStr(job->fd.proc_name))));
```

Comment:
I think this could be a problem, if there is a custom job on the hypertable, it would error out instead of ignoring the job entry.

Reply:
Well, the above code expects that the earlier types SHOULD produce a valid "user_fctx" field which it won't if there is a custom job. So this error is good to keep in place till that `else` part gets added here.

Reply:
any other comments @mfundul ?

Reply:
Two observations:
1.) This is actually a valid compiler warning (as in a bug), so it needs to be handled somehow. We also can not skip on a list element, as we need to return a datum. We could potentially return a dummy jsonb datum.

2.) `show_policies()` is in the experimental schema, so we should be free to handle custom jobs at a later date.